xmovie/core.py

```diff
@@ -304,9 +304,7 @@ def __init__(
         if input_check:
             if isinstance(self.data, xr.Dataset):
                 raise ValueError(
-                    "xmovie presets do not yet support the input of xr.Datasets. \
-                In order to use datasets as inputs, set `input_check` to False. \
-                Note that this requires you to manually set colorlimits etc."
+                    "xmovie presets do not yet support the input of xr.Datasets.\nIn order to use datasets as inputs, set `input_check` to False.\nNote that this requires you to manually set colorlimits etc."
```

Comment:
should we modify this to 'fully supports xr.Datasets'? Might be more true to the code? Just a suggestion